saleor/checkout/core.py

```diff
@@ -28,6 +29,8 @@ def __init__(self, cart, user, tracking_code):
         self.tracking_code = tracking_code
         self.user = user
         self.discounts = cart.discounts
+        self.cached_shipping_method = None
```

Comment:
@maciekjablonski Tried it, but cached_property does not work with property setters.